src/components/LandingPage/Header.tsx

```diff
@@ -46,90 +43,219 @@ const LogoContainer = styled.div`
     font-size: 16px;
     line-height: 19px;
     cursor: pointer;
+    padding-bottom: 2rem;
+    @media (max-width: 475px) {
+        padding-bottom: 0;
+    }
+`;
+const SubMenu = styled.div`
+    position: absolute;
+    display: ${(prop) => (!prop.show ? 'none' : 'flex')};
+    margin-top: 1rem;
+    flex-direction: column;
+    padding: 1.25rem;
+    border: 1px solid #d6d6d6;
+    border-radius: 6px;
+    background: #ffffff;
```

Comment:
Everywhere that has a color value should be referencing a variable from `App.css`.

Reply:
Will change that!